src/node.js

```diff
@@ -393,15 +399,22 @@
     function TickObject(c) {
       this.callback = c;
       this.domain = process.domain || null;
+      this._args = undefined;
     }
 
+    TickObject.prototype._setArgs = function _setArgs() {
+      this._args = arguments;
```

Comment:
It's 33% faster to do it without arguments materialization:

```js
  this.args = new Array(arguments.length);
  for (var i = 0; i < arguments.length; ++i) {
    this.args[i] = arguments[i];
  }
```

Could also define the most often used amount of arguments to avoid arguments adaptor for those cases.